prow/cmd/crier/main.go

```diff
@@ -210,6 +237,31 @@ func main() {
 		}
 	}
 
+	if o.webhookWorkers > 0 {
+		if cfg().WebhookReporterConfigs == nil {
+			logrus.Fatal("webhookreporter is enabled but has no config")
+		}
+		webhookConfig := func(refs *prowapi.Refs) config.WebhookReporter {
+			return cfg().WebhookReporterConfigs.GetWebhookReporter(refs)
+		}
+
+		if o.webhookKeyFile == "" {
+			logrus.Fatal("webhookreporter is enabled but has no key file")
+		}
```

Comment:
This will never be the case, right? It is already validated prior to this execution.

Reply:
That's my impression as well; I left it since it was in the Slack counterpart.

Reply:
or maybe I've come up with it myself? OK removing.